src/crypto/openssl/cose_sign.cpp

```diff
@@ -83,9 +79,83 @@ namespace
 
 namespace ccf::crypto
 {
+  std::optional<int> key_to_cose_alg_id(ccf::crypto::PublicKey_OpenSSL& key)
+  {
+    const auto cid = key.get_curve_id();
+    switch (cid)
+    {
+      case ccf::crypto::CurveID::SECP256K1:
```

Comment:
Is it right that K1 and R1 have the same alg value?

Reply:
DOne